Microsoft.NET.Build.Containers/Microsoft.NET.Build.Containers.csproj

```diff
@@ -1,34 +1,20 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net7.0</TargetFramework>
-    <ImplicitUsings>enable</ImplicitUsings>
+    <TargetFrameworks>net7.0;net472</TargetFrameworks>
+    <ImplicitUsings Condition="'$(TargetFramework)' != 'net472'">enable</ImplicitUsings>
     <Nullable>enable</Nullable>
-    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
-
-    <IsPackable>true</IsPackable>
+    <LangVersion>preview</LangVersion>
 
     <TargetsForTfmSpecificBuildOutput>
       $(TargetsForTfmSpecificBuildOutput);CopyProjectReferencesToPackage
     </TargetsForTfmSpecificBuildOutput>
 
-    <BuildOutputTargetFolder>tasks</BuildOutputTargetFolder>
-
     <!-- Tell the SDK to generate a deps.json file -->
     <GenerateDependencyFile>true</GenerateDependencyFile>
 
-    <!-- MSBuild tasks shouldn't be referenced. This is by design. -->
-    <NoWarn>NU5100;NU5128</NoWarn>
-    <Authors>Microsoft</Authors>
-    <Company>Microsoft</Company>
-    <Description>Tasks and targets to natively publish .NET applications as containers.</Description>
-    <Copyright>&#169; Microsoft Corporation. All rights reserved.</Copyright>
-    <PackageLicenseExpression>MIT</PackageLicenseExpression>
-    <PackageProjectUrl>https://github.com/dotnet/sdk-container-builds</PackageProjectUrl>
-    <RepositoryUrl>https://github.com/dotnet/sdk-container-builds</RepositoryUrl>
-    <RepositoryType>git</RepositoryType>
-    <PackageTags>containers;docker;Microsoft.NET.Build.Containers</PackageTags>
-    <PackageReadmeFile>README.md</PackageReadmeFile>
+    <!-- Allow the packaging project to use the name `Microsoft.NET.Build.Containers` for the nuget package -->
+    <Packageid>.</Packageid>
```

Comment:
Does this project get packed directly anymore? if not, this PackageId may be able to be removed entirely.

Reply:
If this isn't in here then I get `C:\Program Files\dotnet\sdk\7.0.100-preview.7.22377.5\NuGet.targets(132,5): error : Ambiguous project name 'Microsoft.NET.Build.Containers'. [C:\src\
containers\System.Containers.sln]` during builds :/

Reply:
We could theoretically rename stuff, but I figured this was easier.